descwl_shear_sims/sim.py

```diff
@@ -115,6 +117,8 @@ def make_sim(
     draw_method: string
         Draw method for galsim objects, default 'auto'.  Set to
         'phot' to get poisson noise.  Note this is much slower.
+    theta0: float
+        The rotation angle of the exposure, in units of radians
```

Comment:
Should be "The rotation angle of the objects, in units of radians" rather than exposure

Reply:
done!